LoopKit/InsulinKit/DoseStore.swift

```diff
@@ -33,6 +33,63 @@ public enum DoseStoreResult<T> {
     case failure(DoseStore.DoseStoreError)
 }
 
+public protocol DoseStoreProtocol {
```

Comment:
This protocol was added so it's possible to create mock stores to use with LoopDataManager

Reply:
Again, I think keeping this in Loop is better; let's discuss if you feel there is a good reason to keep it here.